torch/utils/cpp_extension.py

```diff
@@ -1724,3 +1740,10 @@ def _is_cuda_file(path):
     if IS_HIP_EXTENSION:
         valid_ext.append('.hip')
     return os.path.splitext(path)[1] in valid_ext
+
+def _to_absolute_path(paths):
```

Comment:
The name here implies that this is a pure function, but actually it mutates the list in place. A more descriptive name would be better.